keps/1136-provisioning-request-support/README.md

```diff
@@ -122,12 +122,21 @@ workload pods - [KEP #1145](https://github.com/kubernetes-sigs/kueue/blob/main/k
 the provisioning request should also be deleted (the last one can be achieved via
 OwnerReference).
 
-* Retry ProvisioningRequests with respect to the `RetryConfig` configuration in
+* Retry ProvisioningRequests with respect to the `RetryStrategy` configuration in
 the `ProvisioningRequestConfig`. For each attempt a new provisioning request is
-created with the suffix indicating the attempt number. The corresponding admission
-check will remain in the `Pending` state until the retries end. The max number
-of retries is 3, and the interval between attempts grows exponentially, starting
-from 1min (1, 2, 4 min).
+created with the suffix indicating the attempt number. The corresponding AdmissionCheck will change Workload's `.status.requeueState` accordingly, and change its status to `Retry` state until the Workload is requeued. After the Workload is requeued the workload controller should change the AdmissionCheck's status to `Pending`.
+Unless a user configures otherwise configuration is as follows:
+  - The max number of retries is 3,
+  - The interval between attempts grows exponentially, starting
+from 1min (1, 2, 4 min),
+  - Workloads gets requeued (and the allocated quota is released) after each failure of ProvisioningRequest.
+
+Workload gets requeued in a similar way to [WaitForPodsReady](https://github.com/kubernetes-sigs/kueue/tree/main/keps/349-all-or-nothing) mechanism.
+When AdmissionCheck is in `Retry` state, the workload controller evicts the Workload with `WorkloadRequeued=False` condition with `AdmissionCheck` as a reason.
+After the backoff period, the Workload is requeued, enters the queue again, and begins a new admission cycle.
+
+Additionally, we introduce the feature gate `KeepQuotaForProvReqRetry`. If the feature gate is enabled the Workload with failed ProvisioningRequest
```

Comment:
Maybe just add a sentence to introduce the motivation: Additionally, to enable the previous behavior of keeping the quota for use users in 0.9 we introduce the ( .... ) The feature gate is deprecated and will be removed in 0.10 unless we get feedback from users indicating that the old behavior is needed.

Reply:
Done